2022-02-02 03:13:24,587 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from /root/.config/wandb/settings
2022-02-02 03:13:24,588 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from /work/wandb/settings
2022-02-02 03:13:24,588 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-02 03:13:24,588 INFO    MainThread:71 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-02-02 03:13:24,588 INFO    MainThread:71 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-02 03:13:24,588 INFO    MainThread:71 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-02 03:13:24,589 INFO    MainThread:71 [wandb_init.py:_log_setup():386] Logging user logs to /work/wandb/run-20220202_031323-xgb_depth_test/logs/debug.log
2022-02-02 03:13:24,589 INFO    MainThread:71 [wandb_init.py:_log_setup():387] Logging internal logs to /work/wandb/run-20220202_031323-xgb_depth_test/logs/debug-internal.log
2022-02-02 03:13:24,589 INFO    MainThread:71 [wandb_init.py:_jupyter_setup():336] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f1b3031b1d0>
2022-02-02 03:13:24,590 INFO    MainThread:71 [wandb_init.py:init():420] calling init triggers
2022-02-02 03:13:24,590 INFO    MainThread:71 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-02-02 03:13:24,590 INFO    MainThread:71 [wandb_init.py:init():442] re-initializing run, found existing run on stack: xgb_depth_test
2022-02-02 03:13:24,602 INFO    MainThread:71 [wandb_run.py:finish():1468] finishing run cz4041/test-project/xgb_depth_test
2022-02-02 03:13:24,602 INFO    MainThread:71 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-02-02 03:13:24,606 INFO    MainThread:71 [wandb_run.py:_restore():1769] restore
2022-02-02 03:13:25,457 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:13:25,715 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:13:26,603 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 17163
}

2022-02-02 03:13:26,718 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 17163
}

2022-02-02 03:13:26,832 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5123
  total_bytes: 17163
}

2022-02-02 03:13:26,935 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17163
  total_bytes: 17163
}

2022-02-02 03:13:27,037 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17163
  total_bytes: 17163
}

2022-02-02 03:13:27,142 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17163
  total_bytes: 17163
}

2022-02-02 03:13:27,245 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17163
  total_bytes: 17163
}

2022-02-02 03:13:27,347 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17163
  total_bytes: 17163
}

2022-02-02 03:13:27,564 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17163
  total_bytes: 17163
}

2022-02-02 03:13:27,894 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17163
  total_bytes: 17163
}
local_info {
}

2022-02-02 03:13:29,549 INFO    MainThread:71 [wandb_run.py:_append_files():2194] logging synced files
2022-02-02 03:13:29,570 INFO    MainThread:71 [wandb_init.py:init():471] starting backend
2022-02-02 03:13:29,570 INFO    MainThread:71 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-02 03:13:29,574 INFO    MainThread:71 [backend.py:ensure_launched():219] starting backend process...
2022-02-02 03:13:29,585 INFO    MainThread:71 [backend.py:ensure_launched():225] started backend process with pid: 1756
2022-02-02 03:13:29,589 INFO    MainThread:71 [wandb_init.py:init():480] backend started and connected
2022-02-02 03:13:29,603 INFO    MainThread:71 [wandb_run.py:_label_probe_notebook():921] probe notebook
2022-02-02 03:13:29,603 INFO    MainThread:71 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-02 03:13:29,603 INFO    MainThread:71 [wandb_init.py:init():550] updated telemetry
2022-02-02 03:13:29,608 INFO    MainThread:71 [wandb_init.py:init():581] communicating current version
2022-02-02 03:13:33,495 INFO    MainThread:71 [wandb_init.py:init():586] got version response 
2022-02-02 03:13:33,495 INFO    MainThread:71 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-02-02 03:13:33,663 INFO    MainThread:71 [wandb_init.py:init():624] starting run threads in backend
2022-02-02 03:13:38,683 INFO    MainThread:71 [wandb_run.py:_console_start():1827] atexit reg
2022-02-02 03:13:38,685 INFO    MainThread:71 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2022-02-02 03:13:38,685 INFO    MainThread:71 [wandb_run.py:_redirect():1738] Wrapping output streams.
2022-02-02 03:13:38,690 INFO    MainThread:71 [wandb_run.py:_redirect():1762] Redirects installed.
2022-02-02 03:13:38,690 INFO    MainThread:71 [wandb_init.py:init():651] run started, returning control to user process
2022-02-02 03:13:38,693 INFO    MainThread:71 [wandb_run.py:_config_callback():966] config_cb None None {'boosting_type': 'dart', 'objective': 'multi:softmax', 'num_class': 9, 'max_depth': 2, 'min_child_weight': 20, 'gamma': 1, 'subsample': 0.8, 'colsample_bytree': 0.7, 'tree_method': 'hist', 'eval_metric': 'mlogloss', 'eta': 0.04, 'alpha': 1, 'verbose': 2}
2022-02-02 03:13:38,915 INFO    MainThread:71 [wandb_init.py:_pause_backend():299] pausing backend
2022-02-02 03:13:38,915 INFO    MainThread:71 [jupyter.py:save_ipynb():377] looking for notebook: None
2022-02-02 03:14:32,142 INFO    MainThread:71 [wandb_init.py:_resume_backend():308] resuming backend
2022-02-02 03:14:32,145 INFO    MainThread:71 [wandb_init.py:_pause_backend():299] pausing backend
2022-02-02 03:14:32,147 INFO    MainThread:71 [jupyter.py:save_ipynb():377] looking for notebook: None
2022-02-02 03:14:32,173 INFO    MainThread:71 [wandb_init.py:_resume_backend():308] resuming backend
2022-02-02 03:14:32,299 INFO    MainThread:71 [wandb_init.py:_pause_backend():299] pausing backend
2022-02-02 03:14:32,299 INFO    MainThread:71 [jupyter.py:save_ipynb():377] looking for notebook: None
2022-02-02 03:14:34,401 INFO    MainThread:71 [wandb_init.py:_resume_backend():308] resuming backend
2022-02-02 03:14:36,184 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from /root/.config/wandb/settings
2022-02-02 03:14:36,185 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from /work/wandb/settings
2022-02-02 03:14:36,185 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-02 03:14:36,185 INFO    MainThread:71 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-02-02 03:14:36,185 INFO    MainThread:71 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-02 03:14:36,185 INFO    MainThread:71 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-02 03:14:36,185 INFO    MainThread:71 [wandb_init.py:_log_setup():386] Logging user logs to /work/wandb/run-20220202_031434-xgb_depth_test/logs/debug.log
2022-02-02 03:14:36,186 INFO    MainThread:71 [wandb_init.py:_log_setup():387] Logging internal logs to /work/wandb/run-20220202_031434-xgb_depth_test/logs/debug-internal.log
2022-02-02 03:14:36,186 INFO    MainThread:71 [wandb_init.py:init():420] calling init triggers
2022-02-02 03:14:36,186 INFO    MainThread:71 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-02-02 03:14:36,186 INFO    MainThread:71 [wandb_init.py:init():442] re-initializing run, found existing run on stack: xgb_depth_test
2022-02-02 03:14:36,190 INFO    MainThread:71 [wandb_run.py:finish():1468] finishing run cz4041/test-project/xgb_depth_test
2022-02-02 03:14:36,211 INFO    MainThread:71 [jupyter.py:save_history():458] saving 22 cells to _session_history.ipynb
2022-02-02 03:14:36,238 INFO    MainThread:71 [wandb_run.py:_config_callback():966] config_cb ('_wandb', 'session_history') code/_session_history.ipynb None
2022-02-02 03:14:36,312 INFO    MainThread:71 [jupyter.py:save_ipynb():377] looking for notebook: None
2022-02-02 03:14:36,313 INFO    MainThread:71 [wandb_init.py:_jupyter_teardown():318] cleaning up jupyter logic
2022-02-02 03:14:36,313 INFO    MainThread:71 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-02-02 03:14:36,315 INFO    MainThread:71 [wandb_run.py:_restore():1769] restore
2022-02-02 03:14:36,834 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:14:36,940 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:14:37,062 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:14:37,202 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:14:37,309 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 64571
}

2022-02-02 03:14:37,430 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 64571
}

2022-02-02 03:14:37,533 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 64571
  total_bytes: 64571
}

2022-02-02 03:14:37,635 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 64571
  total_bytes: 64571
}

2022-02-02 03:14:37,739 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 64571
  total_bytes: 64571
}

2022-02-02 03:14:37,841 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 64571
  total_bytes: 64571
}

2022-02-02 03:14:37,943 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 64571
  total_bytes: 64571
}

2022-02-02 03:14:38,164 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 64571
  total_bytes: 64571
}

2022-02-02 03:14:38,397 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 64571
  total_bytes: 64571
}
local_info {
}

2022-02-02 03:14:39,963 INFO    MainThread:71 [wandb_run.py:_append_files():2194] logging synced files
